content/ri/faqs.md

```diff
@@ -29,119 +28,119 @@ RedisInsight is a browser based GUI for Redis. It lets you do the following -
-{{% /expand %}}
+{{< /expand >}}
 
 {{% expand "How do I connect to AWS Elasticache?" %}}
 You must install RedisInsight inside your VPC, either on an EC2 instance, or using ECS or Fargate. See [EC2 Installation Instructions]({{< relref "/ri/installing/install-ec2.md" >}})
 {{% /expand %}}
 
-{{% expand "What are the memory limits on your license terms?" %}}
+{{< expand "What are the memory limits on your license terms?" >}}
 Our licensing works on the sum of used memory on the redis instances you have added to RedisInsight. So if your license allows 15GB, you can add 5 redis instances using 3 GB RAM, or 1 redis server using 15 GB.
-{{% /expand %}}
+{{< /expand >}}
 
 {{< /expand-control >}}
 
 ## License and Support
 
 {{< expand-control >}}
 
-{{% expand "How do you calculate used memory?" %}}
+{{< expand "How do you calculate used memory?" >}}
 We run the `info` command and look at used_memory to determine the memory used by redis.
 {{% /expand %}}
 
-{{% expand "What do you mean by perpetual license?" %}}
+{{< expand "What do you mean by perpetual license?" >}}
 Once you buy a license, you can use RedisInsight forever as long as you are within your usage limits. Additionally, you are eligible for software upgrades for 1 year from the date of purchase. After a year, you no longer receive updates, but can continue to use the version of RedisInsight you last downloaded.
-{{% /expand %}}
+{{< /expand >}}
 
-{{% expand "How do I get my license key?" %}}
+{{< expand "How do I get my license key?" >}}
 Once you complete payment, you receive an email with your license key.
-{{% /expand %}}
+{{< /expand >}}
 
 {{< /expand-control >}}
 
 ## Privacy and Security
 
 {{< expand-control >}}
 
-{{% expand "Who has access to my redis servers?" %}}
+{{< expand "Who has access to my redis servers?" >}}
 We provide RedisInsight as a docker container that you install and run on your hardware or cloud account. We do not have any ability to connect to your installation of RedisInsight or look at data within your redis servers.
-{{% /expand %}}
+{{< /expand >}}
 
-{{% expand "How can I secure my RedisInsight installation?" %}}
-We recommend installing HTTPS, whitelisting IP addresses that have access to RedisInsight and/or making it available within your VPN/internal network. Additionally, choose strong passwords for your Redis databases.
-{{% /expand %}}
+{{< expand "How can I secure my RedisInsight installation?" >}}
+We recommend installing HTTPS, whitelisting IP addresses that have access to RedisInsight and/or making it available within your VPN/internal network. Additionally, choose a strong admin password.
```

Comment:
what is "admin password" in this context?

Reply:
@slemeur Can you address this?

Reply:
Remedied in https://github.com/RedisLabs/redislabs-docs/pull/885